modules/dnn/test/test_halide_layers.cpp

```diff
@@ -108,10 +108,10 @@ TEST_P(Convolution, Accuracy)
         throw SkipTestException("");
 
     // TODO: unstable test cases
-    if (backendId == DNN_BACKEND_OPENCV && (targetId == DNN_TARGET_OPENCL || targetId == DNN_TARGET_OPENCL_FP16) &&
-        inChannels == 6 && outChannels == 9 && group == 1 && inSize == Size(5, 6) &&
-        kernel == Size(3, 1) && stride == Size(1, 1) && pad == Size(0, 1) && dilation == Size(1, 1) &&
-        hasBias)
+    std::string paramStr = PrintToString(GetParam());
+    if (paramStr == "([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 2x2, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))" ||
```

Comment:
Please use this guard for "unstable" tests:
```
if (cvtest::skipUnstableTests &&
    ...
)
    throw SkipTestException("Skip unstable test"); 
```

I see these failed tests during this week:
- Linux:
```
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/22, where GetParam() = ([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 1x1, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/27, where GetParam() = ([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 2x2, false, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/31, where GetParam() = ([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 2x2, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/155, where GetParam() = ([6, 9, 1], 5x6, 3x1, 1x1, 0x1, 2x2, false, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/159, where GetParam() = ([6, 9, 1], 5x6, 3x1, 1x1, 0x1, 2x2, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
```
- Windows (OpenCL runtime via Windows Update):
```
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/13, where GetParam() = ([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 2x2, false, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/15, where GetParam() = ([6, 4, 1], 5x6, 3x1, 1x1, 0x1, 2x2, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/77, where GetParam() = ([6, 9, 1], 5x6, 3x1, 1x1, 0x1, 2x2, false, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
[  FAILED  ] Layer_Test_Halide/Convolution.Accuracy/79, where GetParam() = ([6, 9, 1], 5x6, 3x1, 1x1, 0x1, 2x2, true, (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16))
```

Reply:
Disabled all the OpenCL tests with kernel 3x1, stride 1x1 and pad 0x1.